src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py

```diff
@@ -46,10 +46,49 @@
 
 helps['consumption pricesheet'] = """
     type: group
-    short-summary: Inspect the price sheet by Azure subscription within current or specified billing period.
+    short-summary: Inspect the price sheet by Azure subscription within a billing period.
 """
 
 helps['consumption pricesheet show'] = """
     type: command
-    short-summary: Show the price sheet for Azure subscription within a current billing period or specified billing period
+    short-summary: Show the price sheet for Azure subscription within a billing period.
+"""
+
+helps['consumption marketplace'] = """
+    type: group
+    short-summary: Inspect the marketplace by Azure subscription within a billing period.
+"""
+
+helps['consumption marketplace list'] = """
+    type: command
+    short-summary: Show the marketplace for Azure subscription within a billing period.
```

Comment:
Are there other planned commands for this marketplace subgroup?

Reply:
No, there is only one for this subgroup